test/general/test_ids.py

```diff
@@ -57,6 +58,9 @@ def test_duplicate_location_ids(self):
     def test_postgen_datapackage(self):
         """Generates a solo multiworld and checks that the datapackage is still valid"""
         for gamename, world_type in AutoWorldRegister.world_types.items():
+            # because of test/general/__init__.py:56
+            if world_type.hidden:
+                continue
```

Comment:
I think that check should probably move to the relevant code piece and maybe be different - such as `if gamename in ...`

docstring of hidden says "... Does not remove functionality.".